django/core/management/commands/runserver.py

```diff
@@ -129,6 +129,8 @@ def inner_run(self, *args, **options):
         shutdown_message = options.get("shutdown_message", "")
         quit_command = "CTRL-BREAK" if sys.platform == "win32" else "CONTROL-C"
 
+        ipv4_addr = "0.0.0.0" if self.addr == "0" else self.addr
```

Comment:
So if we were going to be strict about var naming, then technically `ipv4_addr` _could_ be an ipv6 addr.  Little things like this are usually better to clean up because you never know what may happen to this code in the future.

Maybe let Felixx or Carlton have the final say but imho it would be cleaner to move the logic away from the dictionary literal:

```suggestion
        if self.addr == "0":
            addr = "0.0.0.0"
        elif self._raw_ipv6:
            addr = "[%s]" % self.addr
        else:
            addr = self.addr
```